### ISSUE TITLE
macOS: TS does not create persistent menu bar drop down entry

### ISSUE THREAD

**Issue by elvisimprsntr** [tailscale/tailscale#9898]
### What is the issue?

TS does not create a persistent menu bar drop down entry when TS is disconnected. 

### Steps to reproduce

1. Install macOS 14.1 RC
2. Install TS app
3. Login to Tailnet
4. Activate TS via TS app or via System -> VPN
5. Observer TS disconnect option available via menu bar
6. Disconnect TS
7. Observe TS VPN entry not present

### Are there any recent changes that introduced the issue?

macOS 14.1 RC
TS 1.51.88

### OS

macOS

### OS version

macOS 14.1 RC

### Tailscale version

TS 1.51.88

### Other software

na

**Comment by agottardo** [comment]
@elvisimprsntr you're on 1.51.188 and not 1.51.88, right?

**Comment by elvisimprsntr** [comment]
@agottardo Correct.  I started :beer: too early today.

**Comment by agottardo** [comment]
It's unclear what the logic behind the displayed items is. Can't find it documented anywhere in the Apple documentation. They haven't touched that menu in almost a decade so I wouldn't be surprised if it was lacking support for network extensions.

How did you set up the `Limetree` VPN in that screenshot? Is it a Cisco IPsec VPN configured via the System Settings app, using this menu item?

<img width="275" alt="Screenshot 2023-10-20 at 11 40 07" src="https://github.com/tailscale/tailscale/assets/9057073/3fc811f5-4a66-47d6-b969-8b308bcfc5b8">

**Comment by elvisimprsntr** [comment]
@agottardo 
Yes, Limtree is an IPSec VPN using the built in add config options.

I have other third party apps that have a setting to enable a persistent menu icon.   Maybe that would be a workaround?
